src/layer/__tests__/retries.ts

```diff
@@ -1,9 +1,9 @@
-import 'jest-setup';
+import '../../../jest-setup';
 import axios from 'axios';
 import MockAdapter from 'axios-mock-adapter';
 
 import { constructFixtureFindTiles } from './fixtures.findTiles';
-import { invalidateCaches } from 'src/utils/Cache';
+import { invalidateCaches } from '../../utils/Cache';
```

Comment:
Since `invalidateCaches` is exported in `src/index.ts`, could it be imported from there?

Reply:
Changed

Reply:
Resolving